tests/api/data/summaries.test.js

```diff
@@ -37,6 +38,21 @@ describe('summaries', () => {
       summaryData.lang.should.equal('en')
     })
 
+    it('should detect summary language', async () => {
```

Comment:
some red tests

```
should return summaries:
     TypeError: Cannot read properties of undefined (reading 'key')
      at Context.<anonymous> (file:///home/vinny/inventaire/docker/inventaire/tests/api/data/summaries.test.js:32:19)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

Reply:
@jum-s this might be caused by the change cache data format, can you try to run the tests in non-quick mode (or just run `./scripts/tests/api/delete_tests_databases.sh`)  to reset the leveldb before running the tests?

Reply:
yep, thanks

Reply:
Most tests passes now after resetting level db correctly (which script `delete-databases` didn't do properly (see fix cabf38154)).
But there is still:

```
  12 passing (7s)
  1 failing

  1) summaries
       openlibrary
         should ignore unknown ISBNs:
     request error: invalid uri: expected uri, got  (string)
```

Reply:
Two problems there:
- `existsOrCreate` was silencing the actual error, fixed by 924c46bc2
- This test uses an ISBN considered valid only in the most recent versions of `isbn3`, which was updated in 424824f59 but we don't have a hook to make other developers rerun `npm install` so you didn't get the update: just running `npm install` should fix the issue

Reply:
What about having a warning with a git hook: 

`.git/hooks/post-checkout`

> invoked when a git checkout is run after having updated the worktree


```sh
CHANGED=`git diff $1 $2 --stat package-lock.json`
if [[ -n $CHANGED ]]
then
    echo "△  Warning: package.json has changed."
    echo "△  You may want to run 'npm install'."
fi
```